docs/distributions/gentoo/installation.md

```diff
@@ -1,89 +1,56 @@
+        * Make sure to follow [this guide](https://wiki.gentoo.org/wiki/Installation_alternatives#Installation_from_non-Gentoo_LiveCDs) along with this guide and the Gentoo Handbook starting at step 2
 * USB-C to USB-A adapter
 * USB Flash Drive
 
 ## Install Procedure
 
-1. Partition your SSD
+1. Follow the [Pre-installation](https://wiki.t2linux.org/guides/preinstall) steps.
 
-    1. Open the Bootcamp installer and follow along until it requests for you to input a Windows ISO. This should clear space for a Linux partition because of removed APFS snapshots.
-    2. Now open up Disk Utility. Make a partition with any format. The amount of space you allocate for this Linux partition will be final, so choose wisely.
+2. Follow the [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Disks).
 
-2. Download a Gentoo ISO and flash it to your USB Flash Drive via [Balena Etcher](https://www.balena.io/etcher/) (you can also use dd)
-3. Ensure that Secure Boot is disabled
+    1. Instead of making a new EFI partition for Gentoo, you should instead mount `/dev/nvme0n1p1` to `/mnt/gentoo/boot/efi`.
+    2. Make sure that you have deleted the partition you made ahead of time, or else you'll lose important data.
 
-    1. Follow the instructions from [this article](https://support.apple.com/en-us/HT208198)
-    2. Once in the Startup Security Utility, set secure boot to no security and enable external boot.
+3. Continue through the Handbook until chapter "Configuring the kernel".
 
-4. Boot into the Live USB enviornment
-
-    1. Plug in the USB Flash Drive into your computer
-    2. Startup while holding the Option key, this will put you in the macOS Startup Manager
-    3. Select the orange EFI Boot option and press enter to boot into it. (If you're using a Ubuntu Live environment, then make sure to select the orange EFI Boot option all the way to the right)
-
-5. Follow the [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Disks)
-
-    1. Instead of making a new EFI partition for Gentoo, you should instead mount `/dev/nvme0n1p1` to `/mnt/gentoo/boot/efi`
-    2. Make sure that you have deleted the partition you made ahead of time, or else you'll lose important data
-
-6. Continue through the Handbook until chapter "Configuring the kernel"
-
-    1. In order to have the Linux Kernel support your device the most, you'll have to build from a modified Linux Kernel source tree.
-    2. Git clone [the t2linux kernel source tree](https://github.com/t2linux/kernel) to `/usr/src`
-        * Make sure to git checkout the version you want. For example, if you want kernel version v5.16.17, you would checkout tag `t2-v5.16.17`.
-    3. Run these commands to symlink the kernel to `/usr/src/linux`:
+    1. In order to build support for T2 MacBooks in the kernel, you'll have to apply a patchset and configure the kernel manually.
```

Comment:
I would make this an unorganized list or merge this in with the description for step 3 because this instruction isn't an instruction it's just explaining what's going to happen in this step.